huaweicloud/resource_huaweicloud_compute_instance.go

```diff
@@ -1030,13 +1036,18 @@ func resourceComputeInstanceV2Update(ctx context.Context, d *schema.ResourceData
 		return diag.Errorf("error creating compute V1.1 client: %s", err)
 	}
 
-	var updateOpts servers.UpdateOpts
+	var updateOpts cloudservers.UpdateOpts
 	if d.HasChange("name") {
```

Comment:
you can use `d.HasChanges("name", "description")` to wrap it

Reply:
fixed